packages/firestore/lite/src/api/reference.ts

```diff
@@ -382,6 +402,31 @@ export class CollectionReference<T = firestore.DocumentData> extends Query<T>
     return this._query.path.canonicalString();
   }
 
+  get parent(): DocumentReference<firestore.DocumentData> | null {
+    const parentPath = this._path.popLast();
+    if (parentPath.isEmpty()) {
+      return null;
+    } else {
+      return new DocumentReference(
+        this.firestore,
+        /* converter= */ null,
```

Comment:
nit: Can you standardize the constructor argument order for `DocumentReference` and `CollectionReference`?

Reply:
I missed this in an earlier review when you made the converter nullable, but the `converter` in the `CollectionReference` constructor can be `readonly`.

Reply:
Done (changed CollectionReference to match). As for `readonly`, that is done in the parent class Query and we rely on the Query's converter property here.